charts/shopware/templates/database.yaml

```diff
@@ -39,7 +39,7 @@ spec:
       {{- end }}
     volumeSpec:
       persistentVolumeClaim:
-        storageClassName: {{ .Values.percona.database.storageClassName | default "" | quote }}
+        storageClassName: {{ .Values.percona.database.storageClassName | quote }}
```

Comment:
Is `storageClassName` required? If not, I think we should not set it and let Kubernetes use the default storage class.

Reply:
That's done when you set it to `""`, I found that easier then do a check for the value. So just undo the change and done